### PR TITLE
Update

### PR DESCRIPTION
**Pull request by tavoweb**
pull lithuanian language

### CODE REVIEW

File: lib/translations_manager/locales.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ module TranslationsManager
     'it',
     # 'ja',
     'ko',
+    'lt',
```
Comment (tavoweb) [inline lib/translations_manager/locales.rb:24]: Its like our neibors latvians 👍

**Comment by nlalonde** [comment]
Latvian is not translated enough on Transifex for us to pull it yet. For example, the core server.yml file is only 3.2% translated. 😢

**Comment by tavoweb** [comment]
I aded Lithuanian language not latvian ...

**Comment by tavoweb** [comment]
its 46 + % translated

**Pull request closed without merging**